src/flow_solver/flow_solver_cases/periodic_turbulence.cpp

```diff
@@ -329,6 +329,9 @@ void PeriodicTurbulence<dim, nstate>::compute_and_update_integrated_quantities(D
     // Initialize the maximum local wave speed to zero; only used for adaptive time step
     if(this->all_param.flow_solver_param.adaptive_time_step == true) this->maximum_local_wave_speed = 0.0;
 
+    // Initialize the maximum local wave speed to zero; only used for adaptive time step
+    if(this->all_param.flow_solver_param.error_adaptive_time_step == true) this->maximum_local_wave_speed = 0.0;
+
```

Comment:
Since this if clause does the same thing as the previous one it might be better to combine the statement.